csrc/nv_internal/tensorrt_llm/thop/thUtils.h

```diff
@@ -16,36 +16,53 @@
 
 #pragma once
 
-#include <ATen/cuda/CUDAContext.h>
-#include <cuda_fp16.h>
-#include <cuda_runtime.h>
+#include <tvm/ffi/container/tensor.h>
 
```

Comment:
thUtils means torch utilities, maybe we can remove this file and move useful functions/macros to `tvm_ffi_utils.h` instead?

Reply:
renamed. actually these macros are only used here, so I let them here though.

Reply:
np, all of them will be removed in #1655